taichi/python/export_lang.cpp

```diff
@@ -424,15 +424,6 @@ void export_lang(py::module &m) {
       .def("serialize", &ExprGroup::serialize);
 
   py::class_<Stmt>(m, "Stmt");
-  py::class_<Program::KernelProxy>(m, "KernelProxy")
-      .def(
-          "define",
-          [](Program::KernelProxy *ker,
-             const std::function<void()> &func) -> Kernel * {
-            py::gil_scoped_release release;
```

Comment:
@lin-hitonami This `gil_scoped_release` was accidentally removed in the refactor? Would you mind adding it back? 
Thanks @strongoier for catching this!